### DISCUSSION TITLE
What's the different between `jax.random.PRNGKey` and `jax.random.key`?

### DISCUSSION THREAD

**Discussion by lanchihi** [jax-ml/jax#19392] [Q&A] [answered]
Hi, I found there is a new api `jax.random.key` has been add in the latest version(I use 0.4.9 berfore), what's the different between `jax.random.PRNGKey` and `jax.random.key`? Will `jax.random.PRNGKey` be deprecated in the future version?

**Comment by jakevdp** [comment, marked as answer]
In short, `jax.random.key` produces new-style typed keys, `jax.random.PRNGKey` produes older-style raw keys, and will eventually be deprecated. For more, see https://jax.readthedocs.io/en/latest/jep/9263-typed-keys.html

**Reply by lanchihi** [reply]
Thanks for your information.
